### ISSUE TITLE
Input method panel accidently shown

### ISSUE THREAD

**Issue by Cyrillya** [bluemagic123/tModLoader#2571]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

[Input method](https://en.wikipedia.org/wiki/Input_method) shows when it shouldn't. Really annoying.
![2022-06-18](https://user-images.githubusercontent.com/35227653/174435471-9714a08d-ad52-4806-b990-efa16bae06a5.png)
We can play normally though if we turn on the Caps Lock, this way the IME never shows up. However, I wonder why it happens in tModLoader 1.4, but works properly in vanilla Terraria. Also, it will be better if it is really fixed.

### Steps to reproduce

1. Install an input method to your computer. Maybe [this](https://answers.microsoft.com/en-us/windows/forum/all/traditional-chinese-pinyin-keyboard-in-window-10/9353f1ee-589a-48c8-ab64-0748722eb79c?page=1) tutorial helps?
2. Select the input method you just installed.
3. Open tModLoader 1.4 and just randomly click some keys

### Expected Behavior

For us Chinese, we use [input methods](https://en.wikipedia.org/wiki/Input_method) to input Chinese characters
Terraria has a custom IME panel system. For example, the following code in Main.DrawPlayerChat()
```
string compositionString = Platform.Get<IImeService>().CompositionString;
if (compositionString != null && compositionString.Length > 0)
	list.Add(new TextSnippet(compositionString, new Microsoft.Xna.Framework.Color(255, 240, 20)));
```
```
if (drawingPlayerChat && array != null) {
	Vector2 stringSize = ChatManager.GetStringSize(FontAssets.MouseText.Value, array, Vector2.Zero);
	DrawWindowsIMEPanel(new Vector2(88f, screenHeight - 30) + new Vector2(stringSize.X + 10f, -6f));
}
```
It will be like this in game, at least in vanilla Terraria
![image](https://user-images.githubusercontent.com/35227653/174435595-51d114fe-bde2-4088-a999-8fd5f45a73d2.png)
Then we can press the number to select the Chinese character(s) we want from the panel

In vanilla Terraria and tModLoader 1.3, the panel only shows when the game wants you to input something. For example, it only shows up when you chat, edit the name of a chest, or write a sign.

### Actual Behavior

In tModLoader 1.4, the system's IME panel shows whenever you press a key. It just simply thinks you are editing a sign while playing tModLoader.
Also, the method Main.DrawWindowsIMEPanel() becomes completely useless, since the game just uses system IME panel in the bottom right corner, like this.
![2022-06-18](https://user-images.githubusercontent.com/35227653/174435471-9714a08d-ad52-4806-b990-efa16bae06a5.png)
(The [Microsoft Pinyin IME](https://en.wikipedia.org/wiki/Microsoft_Pinyin_IME) shows up because I'm using it)

### Reproduction frequency

Always

### Additional Information

It has never been an issue until this: #1086, when tModLoader became 64-bit and started showing console.
Interestingly, the 1.3 unofficial 64-bit tModLoader also has this problem.

So maybe it's FNA's fault?
Or the system consider tML a console that requires input?
![image](https://user-images.githubusercontent.com/35227653/174436261-318256f0-536d-463a-b820-5caa47e590b0.png)

**Comment by JavidPack** [comment]
Thought: This might be a lost patch or unused compiler directive

**Comment by HydroH** [comment]
.NET 6.0 has removed System.Windows.Forms support for non-winform applications, which is the reason part of IME-related code was removed.

**Comment by JavidPack** [comment]
@Chicken-Bones closable?

**Comment by Chicken-Bones** [comment]
Fixed in 1.4.4, improvements coming in 1.4.5
